### ISSUE TITLE
Fix Sphinx build warnings in 3.7 branch

### ISSUE THREAD

**Issue by ccwang002** [python/python-docs-zh-tw#1]
Current output of `make all` under 3.7 branch:

```bash
...
loading translations [zh_TW]... done
building [mo]: all of 471 po files
WARNING: reading error: cpython/Doc/locales/zh_TW/LC_MESSAGES/library/contextvars.po, expected only letters, got ''        
WARNING: reading error: cpython/Doc/locales/zh_TW/LC_MESSAGES/whatsnew/3.7.po, expected only letters, got ''               
WARNING: reading error: cpython/Doc/locales/zh_TW/LC_MESSAGES/library/dataclasses.po, expected only letters, got ''        
WARNING: reading error: cpython/Doc/locales/zh_TW/LC_MESSAGES/c-api/contextvars.po, expected only letters, got '' 
...
reading sources... [100%]
cpython/Doc/library/dis.rst:11: WARNING: inconsistent term references in translated message. original: [':mod:`dis`', ':term:`bytecode`'], translated: [':mod:`dis`模組支援反組譯分析 CPython :term:`bytecode`']
cpython/Doc/library/statistics.rst:2: WARNING: Block quote ends without a blank line; unexpected unindent.
cpython/Doc/tutorial/introduction.rst:354: WARNING: inconsistent term references in translated message. original: [':ref:`f-strings`'], translated: []
...
build succeeded, 7 warnings.
```

**Comment by adrianliaw** [comment]
@ccwang002 麻煩先不修正 tutorial/introduction 裡面的 warning，這個可以用來讓新手熟悉 GitHub Flow

**Comment by adrianliaw** [comment]
Fixed at fa045262e15d
